web/theme.dart

```diff
@@ -0,0 +1,63 @@
+import 'dart:html';
+
+void init() {
+  var theme = document.getElementById('theme') as InputElement;
+
+  var elementB = document.body;
+
+  theme.addEventListener('change', (event) {
```

Comment:
To keep things a little simpler, we can say something here like, `if (elementB == null) return;`